src/main/java/cubicchunks/converter/headless/command/commands/ConverterNameCommand.java

```diff
@@ -24,23 +24,20 @@
 package cubicchunks.converter.headless.command.commands;
 
 import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.StringArgumentType;
 import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.mojang.brigadier.builder.RequiredArgumentBuilder;
 import cubicchunks.converter.headless.command.HeadlessCommandContext;
 
 public class ConverterNameCommand {
     public static void register(CommandDispatcher<HeadlessCommandContext> dispatcher) {
         dispatcher.register(LiteralArgumentBuilder.<HeadlessCommandContext>literal("converter")
-            .then(LiteralArgumentBuilder.<HeadlessCommandContext>literal("Default")
-                .executes((context) -> {
-                    context.getSource().setConverterName("Default");
-                    return 1;
-                })
-            ).then(LiteralArgumentBuilder.<HeadlessCommandContext>literal("Relocating")
-                .executes((context) -> {
-                    context.getSource().setConverterName("Relocating");
-                    return 1;
-                })
-            )
+                .then(RequiredArgumentBuilder.<HeadlessCommandContext, String>argument("converter", StringArgumentType.string())
+                        .executes((context) -> {
+                            context.getSource().setConverterName(context.getArgument("converter", String.class));
+                            return 1;
+                        })
```

Comment:
No. The idea behind registering each one separately was that you could get working autocomplete and more useful error messages. You should register each possibility from the converter registry.

Reply:
alrighty, didn't realize that. will change accordingly.